src/webhooks/webhooks.service.ts

```diff
@@ -61,36 +86,46 @@ export class WebhooksService {
       throw new HttpException('Unable to find user', HttpStatus.BAD_REQUEST);
     }
 
-    const partnerAccessDetails = await this.partnerAccessRepository.findOne({
+    const partnerAccessDetails = await this.partnerAccessRepository.find({
       userId: userDetails.id,
+      active: true,
     });
 
-    if (!partnerAccessDetails) {
+    if (!partnerAccessDetails.length) {
       throw new HttpException('Unable to find partner access code', HttpStatus.BAD_REQUEST);
     }
 
+    //_.maxBy returns the partner access that has the highest number of therapy session remaining
+    let hasFeatureLiveChat = false;
+    const partnerAccess = _.maxBy(partnerAccessDetails, async (paDetails) => {
```

Comment:
ah the logic for a new therapy booking is to use the first partnerAccess that was added, until the sessions are used up, then move on to the second partnerAccess that was applied. 
so we can remove this max functionality, and just get the first one with sessions remaining
```
let partnerAccess = partnerAccesses.find(function (partnerAccess) {
      return partnerAccess.featureTherapy === true && partnerAccess.therapySessionsRemaining > 0;
    });
```

and find `hasFeatureLiveChat` by 
```
let partnerAccess = partnerAccesses.find(function (partnerAccess) {
      return partnerAccess.featureLiveChat === true
    });